nele/__init__.py

```diff
@@ -119,16 +128,17 @@ def send_newsletter(source, config, recipients):
 def main():
     arguments = docopt(__doc__, version='Nele %s' % ___version___)
     source = arguments['<newsletter_source>']
-    config = yaml.load(open(arguments['--config']))
+    config = yaml.load(open(arguments['--config']), Loader=yaml.SafeLoader)
```

Comment:
Ist das dasselbe wie 
```suggestion
    config = yaml.safe_load(open(arguments['--config']))
```
?